controller/share_manager_controller.go

```diff
@@ -632,8 +680,71 @@ func (c *ShareManagerController) cleanupShareManagerService(shareManager *longho
 	return c.ds.DeleteService(shareManager.Namespace, service.Name)
 }
 
+// markShareManagerLeaseDelinquent zeros the acquire time field as a flag that the volume
+// should be fast-tracked for failover away from the current lease-holding node.
+func (c *ShareManagerController) markShareManagerDelinquent(sm *longhorn.ShareManager) error {
+	log := getLoggerForShareManager(c.logger, sm)
+
+	lease, err := c.ds.GetLease(sm.Name)
+	if err != nil && !apierrors.IsNotFound(err) {
+		log.WithError(err).Warnf("Failed to retrieve lease for share manager %v from datastore", sm.Name)
+		return err
+	}
```

Comment:
Why skip NotFound error here? 
The following codes only handles the case that lease exists. The function returns NotFound error if the lease isn't existing.

Reply:
Code has changed. Could you please take a look again?